src/puya/options.py

```diff
@@ -22,3 +22,4 @@ class PuyaOptions:
     debug_level: int = 0
     optimization_level: int = 0
     log_level: LogLevel = LogLevel.info
+    target_avm_version: int = 8
```

Comment:
Prior to this I think we had been assuming v9, so maybe the default should be that?

Also should this result in a `#pragma version ` line in the TEAL?

Reply:
yup, the line which writes the `#pragma` was hardcoded to 8 which is why I chose 8 🤷

Reply:
Oh I see, 8 seems fine then